pkg/server/event_handler.go

```diff
@@ -161,6 +166,7 @@ type PubSubMessage struct {
 // in HTTP requests and calls [Handle] to handle the events.
 //
 // [GCS notifications]: https://cloud.google.com/storage/docs/pubsub-notifications#format
+// Object's metadata change will be included in payload of the notification.
```

Comment:
nit: Can you keep the links at the end of comment?
```suggestion
// in HTTP requests and calls [Handle] to handle the events.
// Object's metadata change will be included in payload of the notification.
//
// [GCS notifications]: https://cloud.google.com/storage/docs/pubsub-notifications#format
```